specification/botservice/resource-manager/Microsoft.BotService/stable/2020-06-02/botservice.json

```diff
@@ -0,0 +1,2550 @@
+        "landingPageUrl": {
+          "type": "string",
+          "description": "The Slack landing page Url"
+        },
+        "redirectAction": {
+          "type": "string",
+          "description": "The Slack redirect action",
+          "readOnly": true
+        },
+        "lastSubmissionId": {
+          "type": "string",
+          "description": "The Sms auth token",
+          "readOnly": true
+        },
+        "registerBeforeOAuthFlow": {
+          "type": "boolean",
+          "description": "Whether to register the settings before OAuth validation is performed. Recommended to True.",
+          "readOnly": true
+        },
+        "isValidated": {
+          "type": "boolean",
+          "description": "Whether this channel is validated for the bot",
+          "readOnly": true
+        },
+        "signingSecret": {
+          "type": "string",
+          "description": "The Slack signing secret."
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "Whether this channel is enabled for the bot"
+        }
+      },
+      "description": "The parameters to provide for the Slack channel.",
+      "required": [
+        "clientId",
+        "clientSecret",
+        "verificationToken",
+        "isEnabled"
+      ]
+    },
+    "LineChannel": {
+      "type": "object",
+      "description": "Line channel definition",
+      "x-ms-discriminator-value": "LineChannel",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Channel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LineChannelProperties",
+          "description": "The set of properties specific to line channel resource"
+        }
+      }
+    },
+    "LineChannelProperties": {
+      "properties": {
+        "lineRegistrations": {
+          "type": "array",
+          "description": "The list of line channel registrations",
+          "items": {
+            "$ref": "#/definitions/LineRegistration"
+          }
+        },
+        "callbackUrl": {
+          "type": "string",
+          "description": "Callback Url to enter in line registration.",
+          "readOnly": true
+        },
+        "isValidated": {
+          "type": "boolean",
+          "description": "Whether this channel is validated for the bot",
+          "readOnly": true
+        }
+      },
+      "description": "The parameters to provide for the Line channel.",
+      "required": [
+        "lineRegistrations"
+      ]
+    },
+    "LineRegistration": {
+      "properties": {
+        "generatedId": {
+          "x-ms-mutability": [
+            "read"
+          ],
+          "type": "string",
+          "description": "Id generated for the line channel registration",
+          "readOnly": true
+        },
+        "channelSecret": {
+          "type": "string",
+          "description": "Secret for the line channel registration"
+        },
+        "channelAccessToken": {
```

Comment:
Is `channelAccessToken` a secret? If yes, it should be marked with `x-ms-secret: true` extension.

Reply:
Fixed it.